server/controllers/amp-page.js

```diff
@@ -71,6 +71,7 @@ function getAndRender(uuid, options) {
 
 			const shareParams = {};
 			article.shareUrl = `${article.webUrl}?${querystring.stringify(shareParams)}`;
+			article.facebookAppId = '121355018262104';
```

Comment:
> But what is a Facebook app? It's a difficult question, because a Facebook app is impossible to describe. One might ask the same about birds. What are birds? We just don't know.